src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -489,8 +489,7 @@ BOOST_FIXTURE_TEST_CASE(chainstatemanager_loadblockindex, TestChain100Setup)
     }
 
     // Note: cs2's tip is not set when ActivateExistingSnapshot is called.
-    Chainstate& cs2 = WITH_LOCK(::cs_main,
-        return chainman.ActivateExistingSnapshot(*assumed_base->phashBlock));
+    Chainstate& cs2 = WITH_LOCK(::cs_main, return chainman.AddChainstate(std::make_unique<Chainstate>(nullptr, chainman.m_blockman, chainman, *assumed_base->phashBlock)));
```

Comment:
in 1107e04328735abe636692b1a0179c6e46613d4e

micro-nit: one of these you moved over to curly brace notation but not the others.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/30214#discussion_r2568629658

> micro-nit: one of these you moved over to curly brace notation but not the others.

Thanks, switched to brace notation in the other places